angularjs-1.5-basics/gulp-tasks/test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
```

Comment:
__[jshint]__
* Use the function form of "use strict".

<!-- Comment by [SideCI](https://sideci.com) -->